plugins/apache-airflow/src/components/DagTableComponent/DagTableComponent.tsx

```diff
@@ -129,9 +130,23 @@ export const DenseTable = ({ dags }: DenseTableProps) => {
   );
 };
 
-export const DagTableComponent = () => {
+type DagTableComponentProps = {
+  dagIds?: string[];
+};
+
+export const DagTableComponent = ({ dagIds }: DagTableComponentProps) => {
   const apiClient = useApi(apacheAirflowApiRef);
+  const [dagsNotFound, setDagsNotFound] = useState<string[]>();
```

Comment:
I feel that useState might be redundant here, why not look at if there are `dagIds` and `data` while rendering instead?

Reply:
I could filter the `dagIds` by looking in the `data`
```javascript
dagIds.filter(id => !data.find(d => d.dag_id === id))
```
I feel like it's less intuitive to read